src/olympia/promoted/management/commands/promote_by_firefox_themes.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from olympia import amo
+from olympia.addons.models import Addon
+from olympia.constants.promoted import LINE
+from olympia.promoted.models import (
+    PromotedAddon,
+)
+from olympia.users.models import UserProfile
+
+
+class Command(BaseCommand):
+    help = 'Give themes with Firefox as author the "By Firefox" badge'
+
+    def handle(self, *args, **options):
+        firefox_user = UserProfile.objects.get(pk=settings.TASK_USER_ID)
+        addons = (
+            Addon.objects.public()
+            .filter(type=amo.ADDON_STATICTHEME, authors=firefox_user)
+            .no_transforms()
+        )
+        for addon in addons:
+            self.stdout.write(f'Promoting {addon.slug}')
```

Comment:
Nit: could wrap the log in the `created` to only show the newly promoted addons in the log. otherwise it will be a huge amount of noise.

Reply:
We should only run the command once, and none of these are promoted already, so it shouldn't matter - but more importantly, I want the output to show which add-ons we considered - if we noticed any one of them is missing the badge, we'll be able to go back to the logs to check it was part of the ones the command looked at.